scripts/gen-github-release.sh

```diff
@@ -156,7 +156,7 @@ Install supported versions of Go, Node.js, and a compiler such as gcc or clang a
 cd agoric-sdk
 git fetch --all
 git checkout $TAG
-git clean -xdf
+git clean -xdf && git submodule foreach --recursive git clean -xdf
 yarn install
```

Comment:
We need to switch to `.bin/agd build` as well.

And there are other changes I had to make in the release notes, some regarding the upgrade handler name, and some due to state-sync changes.